### PR TITLE
SDK release notes

### PR DESCRIPTION
**Pull request by garcialuis**
[SC-83240](https://app.shortcut.com/replicated/story/83240/create-release-notes-for-the-sdk)

This PR:
- adds a page for the Replicated-SDK release notes.
- adds `replicated-sdk-release-notes` workflow to open a PR to update the release notes when there is a new version of the SDK released.

Adding the replicated sdk release notes to the sidebar will be part of a different PR: https://github.com/replicatedhq/replicated-docs/pull/1344
This is so that this workflow can be merged and tested before the page is visible in our docs site.

### CODE REVIEW

**Comment by garcialuis** [comment]
Adding the replicated sdk release notes to the sidebar will be part of a different PR: https://github.com/replicatedhq/replicated-docs/pull/1344
This is so that this workflow can be merged and tested before the page is visible in our docs site.

**Review by sgalsaleh — DISMISSED** [review]

File: .github/workflows/replicated-sdk-release-notes.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    - name: Generate Release Notes
+      id: release-notes
+      env:
+        REPLICATED_SDK_VERSION: ${{ github.event.client_payload.version }}
+      uses: replicatedhq/release-notes-generator@main
+      with:
+        owner-repo: replicatedhq/replicated-sdk
+        head: v$SDK_VERSION
+        title: $SDK_VERSION
```
Comment (cbodonnell) [inline .github/workflows/replicated-sdk-release-notes.yml:24]: Do these need to match the `REPLICATED_SDK_VERSION` name above?

Comment (garcialuis) [reply .github/workflows/replicated-sdk-release-notes.yml:24]: Yes, thank you for catching that. I'll update them.

**Review by cbodonnell — DISMISSED** [review]

**Comment by jonquil2002** [comment]
@garcialuis  This basically looks good, but the file name needs to be added to the sidebar.js items list (under the second sidebar in the list named `// MAIN SIDEBAR`) so that it shows up in the TOC. It can go under the item `release-notes/rn-whats-new`

Let me know if you have questions. Please ping me when you're ready for another review.

**Pull request merged**